com.unity.render-pipelines.high-definition/Documentation~/Compositor-User-Options.md

```diff
@@ -36,9 +36,10 @@ To expose these properties, select a Sub-layer in the **Render Schedule**.
 | **Source Video**| Specifies the [Video Player](https://docs.unity3d.com/ScriptReference/Video.VideoPlayer.html) to use for this Sub-layer.<br />This property is only available for **Video** Sub-layers. |
 | **Source Camera** |Specifies the Camera to use for this Sub-layer. By default, this is set to the main Camera in the Scene.<br />This property is only available for **Camera** Sub-layers.|
-| **Clear Depth**| Specifies whether Unity clears the depth buffer before it draws the contents of this Sub-layer. |
-| **Clear Alpha** |Specifies whether Unity clears the alpha channel before it draws the contents of this Sub-layer. If you enable this property, post-processing only affects pixels drawn by previous Sub-layers. Otherwise, post-processing also affects pixels drawn in previous Composition Layers.|
-| **Clear Color** |Overrides the **Background Type** for this Sub-layer. By default, this has the same value as **Background Type** on the Sub-layer's Camera. To override this value, enable the checkbox then select the new value from the drop-down.|
+| **Clear Depth**| Specifies whether Unity clears the depth buffer before it draws the contents of this Sub-layer. This option is inactive for the first Sub-layer in a stack, since this Sub-layer always clears the depth.|
+| **Clear Alpha** | Specifies whether Unity clears the alpha channel before it draws the contents of this Sub-layer. If you enable this property, post-processing only affects pixels drawn by previous Sub-layers. Otherwise, post-processing also affects pixels drawn in previous Composition Layers.|
+| **Alpha Range** | Controls how steep the transition between the post-processed and plain regions will be. The full [0, 1]  range will provide the smoothest transition, while shortest ranges will provide a steeper transition. This option should be adjusted in case you see outlines around transparent objects when using per-layer post processing. For more information on how the alpha channel is used in the compositor see [Using the Graphics Compositor](Compositor-User-Guide.md).
+| **Clear Color** |Overrides the **Background Type** for this Sub-layer. By default, this has the same value as **Background Type** on the Sub-layer's Camera. To override this value, enable the checkbox then select the new value from the drop-down. This option is only available for the first Sub-layer in a stack. The other sub-layers never clear the color buffer (as required for camera stacking). Setting this option to **None** will leave the color values unitialized when the rendering of the sub-layer starts. This guarantees that the color will not be cleared, but there is no guarantee what there will be already on the buffer when you start drawing.|
```

Comment:
Overrides the **Background Type** for this Sub-layer. By default, this has the same value as **Background Type** on the Sub-layer's Camera. To override this value, enable the checkbox then select the new value from the drop-down. This option is only available for the first Sub-layer in a stack. The other Sub-layers never clear the color buffer (as required for camera stacking). Setting this option to **None** means the color values are uninitialized when the rendering of the sub-layer starts. This guarantees that the compositor does not clear the color, but there is no guarantee of what the contents of the buffer are when you start drawing.